wlm_server/measurement/consumers.py

```diff
@@ -0,0 +1,54 @@
+import json
+from datetime import timedelta
+
+from django.utils import timezone
+from channels.generic.websocket import AsyncWebsocketConsumer
+from asgiref.sync import sync_to_async
+from camel_converter import dict_to_camel
+
+from measurement.models import Measurement
+from measurement.serializers import MeasurementSerializer
+
+class MeasurementConsumer(AsyncWebsocketConsumer):
+    """Consumer for notifying the measurement of a specific channel.
+    
+    Attributes:
+        ch: Target WLM channel.
+        group_name: Name of group it belongs to in the channel layer.
+    """
+
+    # pylint: disable=attribute-defined-outside-init
+    async def connect(self):
+        self.ch = self.scope['url_route']['kwargs']['ch']
+        self.group_name = f'channel_{self.ch}_measurement'
+        await self.channel_layer.group_add(self.group_name, self.channel_name)
+        await self.accept()
+        message = await sync_to_async(self.get_recent_measurements)()
```

Comment:
Let's use [`database-sync-to-async`](https://channels.readthedocs.io/en/latest/topics/databases.html#database-sync-to-async) instead as we discussed it offline.